src/components/popup/tx-item.tsx

```diff
@@ -126,6 +101,112 @@ const SpeedUpButton = ({
   );
 };
 
+const TxTransfers: React.FC<{ transaction: AddressTransactionWithTransfers & BoxProps }> = ({
+  transaction,
+  ...rest
+}) => {
+  return (
+    <>
+      {transaction.stx_transfers.map((stxTransfer, index) => (
+        <StxTransferItem stxTransfer={stxTransfer} parentTx={transaction} {...rest} key={index} />
+      ))}
+    </>
+  );
+};
+
+const TxItemRow: React.FC<{
+  title: string;
+  value: number | string | null;
+  caption?: string;
+  icon: JSX.Element;
+  onClick?: () => Window | null;
+}> = ({ title, caption, value, icon, onClick, ...rest }) => {
```

Comment:
This is really just a preference for readability, but in most places we extract the type to `<nameOfComponet>Props`. I think it'd be helpful to keep to this pattern. What do you think?

```suggestion
interface TxItemRowProps {
  title: string;
  value: number | string | null;
  caption?: string;
  icon: JSX.Element;
  onClick?: () => Window | null; // also what's this type? It doesn't return window or null? Should probs be `void`
}
const TxItemRow: React.FC<TxItemRowProps> = ({ title, caption, value, icon, onClick, ...rest }) => {
```

Reply:
Yes this is better